File: Unidad3/Java/u3p10.java
package com.mycompany.Programa10_U3;
import java.util.Scanner;

public class u3p10 {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese las horas trabajadas: ");
        double horasTrabajadas = scanner.nextDouble();
        System.out.print("Ingrese el pago por hora: $");
        double pagoPorHora = scanner.nextDouble();

        double sueldo = horasTrabajadas * pagoPorHora;
        if (horasTrabajadas > 40) {
            double horasExtras = horasTrabajadas - 40;
            double pagoHorasExtras = horasExtras * (pagoPorHora * 2);
            sueldo += pagoHorasExtras;
        }

        System.out.println("El sueldo semanal es: $" + sueldo);
    }
}

File: Unidad3/C#/u3p10.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Ingrese las horas trabajadas: ");
        double horasTrabajadas = double.Parse(Console.ReadLine());
        Console.Write("Ingrese el pago por hora: $");
        double pagoPorHora = double.Parse(Console.ReadLine());

        double sueldo = horasTrabajadas * pagoPorHora;
        if (horasTrabajadas > 40)
        {
            double horasExtras = horasTrabajadas - 40;
            double pagoHorasExtras = horasExtras * (pagoPorHora * 2);
            sueldo += pagoHorasExtras;
        }

        Console.WriteLine("El sueldo semanal es: $" + sueldo);
    }
}
